src/dsp_tools/utils/xml_upload.py

```diff
@@ -770,7 +770,11 @@ def _upload_resources(
         except BaseError as err:
             err_msg = err.orig_err_msg_from_api or err.message
             print(f"WARNING: Unable to create resource '{resource.label}' ({resource.id}): {err_msg}")
-            logger.warning(f"Unable to create resource '{resource.label}' ({resource.id}).", exc_info=True)
+            logger.warning(
+                f"Unable to create resource '{resource.label}' ({resource.id})\n"
+                f"Resource details:\n{vars(resource)}",
+                exc_info=True,
+            )
```

Comment:
with `vars(object)`, you can print detailed info about the object, especially all class attributes. That's very useful for debugging.